com.unity.mobile.notifications/Runtime/Android/AndroidNotificationChannel.cs

```diff
@@ -54,6 +54,9 @@ public enum LockScreenVisibility
         Public = 1,
     }
 
+    /// <summary>
+    /// The wrapper of the Android notification channel which enables developers to group the notifications by groups.
```

Comment:
```suggestion
    /// The wrapper of the Android notification channel. Use this to group notifications by groups.
```

Unsure what we mean "group by groups", though... Can we replace either "group" with something else? For instance,

> Use this to present notifications in groups.

or something like that?